### PR TITLE
EDSC-3611: Removes legacy services

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

File: static/src/js/util/accessMethods/buildAccessMethods.js [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+  } = collectionMetadata
+
+  const accessMethods = {}
+
+  let harmonyIndex = 0
+
+  const { items: serviceItems = null } = services
+
+  if (serviceItems !== null) {
+    serviceItems.forEach((serviceItem) => {
```
Comment (jaybarra) [inline static/src/js/util/accessMethods/buildAccessMethods.js:31]: Could this `forEach` be split apart into smaller chunks? The first section could be simplified with a `filter`, and the each of the `if` feel like they might be able to be broken out into smaller functions

**Review by jaybarra — CHANGES_REQUESTED** [review]

File: serverless/src/getSavedAccessConfigs/handler.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+    const configs = {}
+
+    accessConfigRecords.forEach((config) => {
+      const {
+        collection_id: collectionId,
+        access_method: accessMethod
+      } = config
+
+      // After EDSC-3638, we are saving the form digest as `formDigest`, but previous access configurations will need to be updated
```
[the remark this answers is not in the document]
Comment (macrouch) [reply serverless/src/getSavedAccessConfigs/handler.js:52]: We don't have versions, so no to a minimum version, it's just before this change we used `form_digest`, and after this change we use `formDigest`. The code right after this comment is accounting for that

File: static/src/js/util/accessMethods/buildAccessMethods.js (hunk 1 above)
Comment (macrouch) [reply static/src/js/util/accessMethods/buildAccessMethods.js:31]: Yes I think pulling the `if` statement's code out into smaller utils would be fine, I'm just not sure if I have time to do it before this has to be merged.

File: static/src/js/util/accessMethods/insertSavedAccessConfig.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+              accessMethods[methodName] = {
+                ...methods[methodName],
+                form,
+                model,
+                rawModel,
+                formDigest
+              }
+            } else {
+              console.log('There was a problem parsing the savedAccessConfig values, using the default form instead.')
+            }
```
Comment (jaybarra) [inline static/src/js/util/accessMethods/insertSavedAccessConfig.js:84]: This is long enough to also warrant being extracted into a separate function.

**Review by jaybarra — APPROVED** [review]

File: serverless/src/requeueOrder/handler.js [hunk 4]
```diff
@@ -60,7 +60,7 @@ const requeueOrder = async (event, context) => {
     } else if (type === 'ECHO ORDERS') {
       // Submits to Legacy Services (CMR) and is often referred to as an
```
Comment (eudoroolivares2016) [inline serverless/src/requeueOrder/handler.js:61]: Is this comment invalid now since legacy-services is no longer getting called for this

**Review by eudoroolivares2016 — APPROVED** [review]

**Pull request merged**